roadmap/implementors-guide/src/node/utility/provisioner.md

```diff
@@ -1,30 +1,42 @@
 # Provisioner
 
-This subsystem is responsible for providing data to an external block authorship service beyond the scope of the [Overseer](/node/overseer.html) so that the block authorship service can author blocks containing data produced by various subsystems.
+Relay chain block authorship authority is governed by BABE and is beyond the scope of the Overseer and the rest of the subsystems. That said, ultimately the block author needs to select a set of backable parachain candidates and other consensus data, and assemble a block from them. This subsystem is responsible for providing the necessary data to all potential block authors.
 
-In particular, the data to provide:
+A major feature of the provisioner: this subsystem is responsible for ensuring that parachain block candidates are sufficiently available before sending them to potential block authors.
 
-- backable candidates and their backings
-- signed bitfields
-- misbehavior reports
-- dispute inherent
-    > TODO: needs fleshing out in validity module, related to blacklisting
+## Provisionable Data
 
-## Protocol
+### Backable Candidates
+
+The block author can choose 0 or 1 backable parachain candidates per parachain; the only constraint is that each backable candidate has the appropriate relay parent. However, the choice of a backable candidate must be the block author's; the provisioner must ensure that block authors are aware of all available backable candidates.
+
+> TODO: "and their backings": why?
+
+### Signed Bitfields
+
+Signed bitfields are attestations from a particular validator about which candidates it believes are available.
+
+> TODO: Are these actually included in the relay chain block, or just used to help decide whether a block is available and therefore a backable candidate?
```

Comment:
Yes, included in the relay-chain block. See `Inclusion` and `InclusionInherent` modules